.gitignore

```diff
@@ -25,12 +25,16 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+app/node_modules
 
 # OSX
 .DS_Store
 
 # App packaged
 dist
-release
```

Comment:
If `release` is no longer used, where are the built folders going? Ideally, they we could build to the same folder. A lot of users of this project already expect builds to be there so changing the directory would not be a  good move.

[Here's how you fix that](https://github.com/amilajack/popcorn-time-desktop/blob/985b85c2615f6245da7480afd3102980194a2ce3/package.json#L63)